usaspending_api/disaster/v2/views/disaster_base.py

```diff
@@ -112,6 +111,22 @@ def last_closed_monthly_submission_dates(self):
     def last_closed_quarterly_submission_dates(self):
         return get_last_closed_submission_date(is_quarter=True)
 
+    @cached_property
+    def final_submissions_query_filters(self):
```

Comment:
Name is a bit misleading. If there wasn't a submission in e.g. pd. 12 for an agency, then you don't want their final or latest ... you want nothing in that case. The "final" refers to the period, not the submission.

Maybe `final_period_submission_query_filters`?